src/modules/Builder/features/AboutApplet/AboutApplet.tsx

```diff
@@ -33,7 +34,22 @@ const commonUploaderProps = {
 };
 
 export const AboutApplet = () => {
-  const { t } = useTranslation();
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation();
+  const { result: appletData } = applet.useAppletData() ?? {};
+  const isNewApplet = useCheckIfNewApplet();
+  const defaults = isNewApplet
+    ? defaultValues
+    : ({
+        displayName: appletData?.displayName ?? '',
```

Comment:
could you pls try to distract values like this:
const {displayName = '', description = '', ... } = appletData;